### PR TITLE
Add cache_column

### PR DESCRIPTION
**Pull request by iHiD**
This knocks 250ms off all the community solution pages for popular exercises.

### CODE REVIEW

**Review by ErikSchierboom — COMMENTED** [review]
I think you'll also need to update this in https://github.com/exercism/website/blob/main/app/commands/solution/publish_iteration.rb, which is called in https://github.com/exercism/website/blob/main/app/controllers/api/solutions_controller.rb#L119 without using `Solution::Publish`.

**Comment by iHiD** [comment]
> I think you'll also need to update this in...

I don't think I do as that code doesn't change the quantity of published solutions, unless I'm misunderstanding.

**Comment by iHiD** [comment]
Happy for you to merge this if you're happy too.

**Review by ErikSchierboom — APPROVED** [review]

**Comment by ErikSchierboom** [comment]
Maybe also update: https://github.com/exercism/website/blob/main/app/controllers/tracks/community_solutions_controller.rb#L11?

**Comment by iHiD** [comment]
Could you do this pls?

File: app/jobs/cache_num_published_solutions_on_exercise_job.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+class CacheNumPublishedSolutionsOnExerciseJob < ApplicationJob
+  def perform(exercise)
```
Comment (ErikSchierboom) [inline app/jobs/cache_num_published_solutions_on_exercise_job.rb:4]: Maybe be explicit about the queue being used? We do that in other jobs too.

```suggestion
  queue_as :default

  def perform(exercise)
```

Comment (iHiD) [reply app/jobs/cache_num_published_solutions_on_exercise_job.rb:4]: Good shout. Thanks.

**Pull request merged**